SU-SWS/stanford_news
# D8CORE-2432: changed the font size for the load more button

# READY FOR REVIEW

# Summary
- Changed the load more button size to su-button-primary--big

# Needed By (Date)
- Thursday

# Urgency
- High

# Steps to Test

1. Pull in this, build.
2. Generate 25+ news
3. Look at the news list page and make sure the button is big.

# Affected Projects or Products
- News, SOE

# Associated Issues and/or People
- D8CORE-2432
# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
